_temp/system-context.puml

```diff
@@ -0,0 +1,53 @@
+@startuml system-context
```

Comment:
I see _temp in the gitignore but these are here.  Is that intentional?

Reply:
Uh, yeah. Let me clean that up ...

Reply:
> I think overall this looks good.
> 
> We may want to move the documentation on PUML to somewhere more general but I think we can leave it in here for now.
> 
> Some of the figures seemed a little superfluous but I guess they are fine.
> 
> I did have a few minor remarks that we may want to correct first.
> 
> Also, I couldn't find a place to add the remark. I think the Kafka is more of a connection than a REST call. But I could be mistaken.

I did a quick check on the kafka client, and it did appear that it at least does support a REST api, whether the python client uses that or not I don't know. In the end, I think the most important attribute of the overall architecture diagram is to show the major components and their inter-dependencies. More detail, accurate, is also useful. Tried to keep in mind people who are and aren't familiar with the project. 
Also, in the end, the purpose of this this diagram in this PR is more exemplary and can certainly be improved. More diagrams are required to capture greater detail for specific areas (e.g. how kafka interacts with it, how sample service interacts with RE, the interaction of import with sample service), and some diagrams may be out of context for the sample service in isolation, and should be in a sample "system" documentation repo.